src/components/Form.js

```diff
@@ -14,7 +14,7 @@ import Loader from 'components/Loader';
 import {ConfirmationView, StartPaymentView} from 'components/PostCompletionViews';
 import ProgressIndicator from 'components/ProgressIndicator';
 import RequireSubmission from 'components/RequireSubmission';
-import {RequireSession} from 'components/Sessions';
+import {SessionModalTracker} from 'components/Sessions';
```

Comment:
```suggestion
import {SessionModalTracker} from 'components/Sessions';
```
maybe `SessionTrackerModal`?